azurerm/internal/services/storage/storage_account_resource.go

```diff
@@ -157,6 +157,13 @@ func resourceStorageAccount() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"is_nfsv3_enabled": {
```

Comment:
we could probably remove the `is_` prefix here?

```suggestion
			"nfsv3_enabled": {
```

Reply:
Yes, I was trying to be consistent as `is_hns_enabled`, whilst I think it makes more sense to name it without the `is` prefix.